src/docker/composebuilder/index.go

```diff
@@ -1,26 +1,42 @@
 package composebuilder
```

Comment:
Changed filename to `index.go`, since that's pretty much what we were treating it as. It always confused me that it was `application.go`